python/ray/data/_internal/block_batching/iter_batches.py

```diff
@@ -274,17 +284,19 @@ def prefetch_batches_locally(
         sliding_window.append(next_block_ref_and_metadata)
         current_window_size += next_block_ref_and_metadata[1].num_rows
 
-    prefetcher.prefetch_blocks([block_ref for block_ref, _ in list(sliding_window)])
+    with stats.iter_wait_s.timer() if stats else nullcontext():
+        prefetcher.prefetch_blocks([block_ref for block_ref, _ in list(sliding_window)])
```

Comment:
`prefetch_blocks` is asynchronous. measuring this doesn't make sense. and I noticed `block_batching.py` already has this issue. 
I'm confused about the exact meaning of `iter_wait_s`. `iter_get_s` should already reflect the time of waiting for the blocks. cc @c21

Reply:
I see, I'll remove this change from this pr anyways since we're not emitting this metric to prometheus anymore

Reply:
Discussed offline with @raulchen, we can remove `iter_wait_s` now.